cider/function/string/StringOps.h

```diff
@@ -44,7 +44,7 @@ namespace StringOps_Namespace {
 
 struct NullableStrType {
   NullableStrType(const std::string& str) : str(str), is_null(false) {}
-  NullableStrType(const std::string_view& sv) : str(sv), is_null(false) {}
+  NullableStrType(const std::string_view sv) : str(sv), is_null(false) {}
```

Comment:
should remove reference here?

Reply:
std::string_view is a read-only view over a string, it only holds a pointer to the first character of the string and its length. It says the best practice to pass string_view is by value